.storybook/codeEditorAddon/codeAddon.js

```diff
@@ -0,0 +1,112 @@
+import { makeDecorator } from '@storybook/addons';
```

Comment:
code editor overrides knob adjustments

Reply:
Unfortunately, I don't think there is an easy fix here and we need to re-think how we handle knobs. Given that we want to additionally support `attributes` (#262) in addition to properties and css, I recommend we disable knobs temporarily while we work on fixing the issue in another PR.